### PR TITLE
Added more info on RTM agent API, including the link.

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by bauerchristina — DISMISSED** [review]
Thank you, Hanna.

File: modules/ROOT/pages/runtime-manager-agent-api.adoc [hunk 1]
```diff
@@ -9,15 +9,33 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-For the majority of use cases, all you'll need to do is install and configure the Runtime Manager agent using xref:installing-and-configuring-runtime-manager-agent.adoc[Install or Update the Runtime Manager agent]
 
-To use the Runtime Manager agent for more than managing the communication between xref:index.adoc[Runtime Manager] and Runtime Manager on-premises, you might find the documents in this section helpful:
+The Runtime Manager agent API enables you to manage your on-premises Mule runtime engine directly using a REST API.
 
+For the majority of use cases, you only need to install and configure the Runtime Manager agent.
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-manager-agent-api.adoc]:
```suggestion
For the majority of use cases, you need only to install and configure the Runtime Manager agent.
```

File: modules/ROOT/pages/runtime-manager-agent-api.adoc [hunk 2]
```diff
@@ -9,15 +9,33 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid
...
+To use the Runtime Manager agent for more than managing the communication between Runtime Manager and on-premises Mule instances, use the Runtime Manager Agent API. 
+
+The Runtime Manager Agent REST API enables you to programmatically access these functions of the agent:
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-manager-agent-api.adoc]: Should we consistently use the full name: the Runtime Manager agent REST API?

Comment (kbostrom) [reply modules/ROOT/pages/runtime-manager-agent-api.adoc]:
```suggestion
To use the Runtime Manager agent for more than managing the communication between Runtime Manager and on-premises Mule instances, use the Runtime Manager Agent API. 

The Runtime Manager agent REST API enables you to programmatically access these functions of the agent:
```

File: modules/ROOT/pages/runtime-manager-agent-api.adoc [hunk 3]
```diff
@@ -9,15 +9,33 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid
...
+** Event tracking
+** Splunk integration
+** ELK integration
+
+For an interactive reference of all the operations that are supported through the API, see https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/mule-agent-plugin-api/minor/1.1/pages/home/[Runtime Manager Agent API] on Exchange.
+
+
+== See Also
+
+* xref:installing-and-configuring-runtime-manager-agent.adoc[Install or Update the Runtime Manager agent]
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-manager-agent-api.adoc]:
```suggestion
* xref:installing-and-configuring-runtime-manager-agent.adoc[Install or Update the Runtime Manager Agent]
```

**Review by kbostrom — DISMISSED** [review]
Edited per scope of request

**Review by kbostrom — APPROVED** [review]
Reapproval

**Pull request merged**